Core/GraphQL/GraphQLFiles/sell_getMyPaymentSources.graphql

```diff
@@ -3,5 +3,8 @@ query getMyPaymentSources($pageSize: Int!, $page: Int! = 1) {
     edges {
       ...paymentSourceDetails
     }
+    nextPage
+    lastPage
+    firstPage
```

Comment:
Could you remove `lastPage` and `firstPage` please? I don't think we need these two fields